### ISSUE TITLE
TODO: README

### ISSUE THREAD

**Issue by kscott-1** [hihg-um/docker-utils#3]
Add a readme for this repo - Not sure if we want to change the format in other repos, but I think it can all be improved

**Comment by thebigcorporation** [comment]
Good point. I think the readmes can almost be cookie cutter. We have a docker-example repo that could be cleaned up as template to avoid oversights like this.I’ll work on it as I get time. Don’t merge this yet, I also have a commit to verify gpg sigs on mount-s3 in the works.On May 22, 2024, at 06:51, Kyle Scott ***@***.***> wrote:﻿
Add a readme for this repo - Not sure if we want to change the format in other repos, but I think it can all be improved
